vcpkg/vcpkg.json

```diff
@@ -42,8 +42,9 @@
     "expat",
     {
       "name": "gdal",
-      "default-features": false,
```

Comment:
Glad to see this! I was hesitating to propose this because I was not sure how works the default-feature/recommended-feature override [here](https://github.com/qgis/QGIS/blob/2db31c8c19fea254a7b2c4318219d1a6bca13968/vcpkg/ports/gdal/vcpkg.json#L65) and [there](https://github.com/qgis/QGIS/blob/2db31c8c19fea254a7b2c4318219d1a6bca13968/vcpkg/ports/gdal/vcpkg.json#L278)

Btw, Is [this](https://github.com/qgis/QGIS/blob/2db31c8c19fea254a7b2c4318219d1a6bca13968/vcpkg/ports/gdal/vcpkg.json#L65) normal ?

So far, QGIS was not able to open Geopackage because sqlite was missing because of this line

Reply:
OK, I understand it better, thanks :+1: